app/domain/authentication/authn_jwt/validate_resource_restrictions.rb

```diff
@@ -0,0 +1,46 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    ValidateResourceRestrictions = CommandClass.new(
+      dependencies: {
+        logger: Rails.logger,
+        # TODO: Dependency for extract_resource_restrictions should be added here
+        # TODO: Dependency for validate_resource_restrictions_configuration should be added here
+        # TODO: Dependency for validate_request_matches_resource_restrictions should be added here
+      },
+      inputs: %i[authentication_parameters]
+    ) do
+      def call
+        @logger.debug(LogMessages::Authentication::ResourceRestrictions::ValidatingResourceRestrictions.new(@role_name))
```

Comment:
The role name should be a the identity from the authentication_parameters

Reply:
Done